WcaOnRails/spec/support/database_cleaner.rb

```diff
@@ -13,10 +13,32 @@
+    FactoryGirl.create(:committee_position, slug: 'team-leader', name: 'Team Leader', description: 'Leader of the software team', team_leader: true, committee_id: @software.id)
+    FactoryGirl.create(:committee_position, slug: 'team-member', name: 'Team Member', description: 'Member of the software team', team_leader: false, committee_id: @software.id)
+    FactoryGirl.create(:committee_position, slug: 'candidate-member', name: 'Candidate Member', description: 'Candidate member of the software team', team_leader: false, committee_id: @software.id)
+    @results = FactoryGirl.create(:committee, slug: 'wca-results-committee', name: 'WCA Results Committee', duties: 'Does results')
+    FactoryGirl.create(:committee_position, slug: 'team-leader', name: 'Team Leader', description: 'Leader of the results team', team_leader: true, committee_id: @results.id)
+    FactoryGirl.create(:committee_position, slug: 'team-member', name: 'Team Member', description: 'Member of the results team', team_leader: false, committee_id: @results.id)
+    @regulations = FactoryGirl.create(:committee, slug: 'wca-regulations-committee', name: 'WCA Regulations Committee', duties: 'Does regulations')
+    FactoryGirl.create(:committee_position, slug: 'team-leader', name: 'Team Leader', description: 'Leader of the regulations team', team_leader: true, committee_id: @regulations.id)
+    FactoryGirl.create(:committee_position, slug: 'team-member', name: 'Team Member', description: 'Member of the regulations team', team_leader: false, committee_id: @regulations.id)
+    @disciplinary = FactoryGirl.create(:committee, slug: 'wca-disciplinary-committee', name: 'WCA Disciplinary Committee', duties: 'Does disciplining')
+    FactoryGirl.create(:committee_position, slug: 'team-leader', name: 'Team Leader', description: 'Leader of the disciplinary team', team_leader: true, committee_id: @disciplinary.id)
+    FactoryGirl.create(:committee_position, slug: 'team-member', name: 'Team Member', description: 'Member of the disciplinary team', team_leader: false, committee_id: @disciplinary.id)
+
+    FactoryGirl.create(:team, slug: 'board-members', name: 'Board Members', description: 'The board team that is part of the WCA board', committee_id: @board.id)
+    FactoryGirl.create(:team, slug: 'oceania', name: 'Oceania', description: 'Delegates for official competitions in the Oceania region.', committee_id: @delegates.id)
+    FactoryGirl.create(:team, slug: 'software-team', name: 'Software Team', description: 'Does software', committee_id: @software.id)
+    FactoryGirl.create(:team, slug: 'results-team', name: 'Results Team', description: 'Posts results', committee_id: @results.id)
+    FactoryGirl.create(:team, slug: 'regulations-team', name: 'Regulations Team', description: 'Regulations', committee_id: @regulations.id)
+    FactoryGirl.create(:team, slug: 'disciplinary-team', name: 'Disciplinary Team', description: 'Disciplinary Committee', committee_id: @disciplinary.id)
```

Comment:
This seems a bit scary to me. Isn't this going to make our tests noticeably slower?